assets/dev/js/editor/data/globals/colors/commands/create.js

```diff
@@ -0,0 +1,23 @@
+import CreateBase from 'elementor-editor/data/globals/base/create-base';
+
+export class Create extends CreateBase {
+	apply( args = {} ) {
+		const { container, setting, title } = args,
+			controls = container.controls;
+
+		let result = false;
+
+		if ( ! controls[ setting ] ) {
+			throw new Error( `Invalid setting: control '${ setting }', not found.` );
+		}
+
+		const id = elementor.helpers.getUniqueID();
+
+		result = $e.data.create( `globals/colors?id=${ id }`, {
```

Comment:
Endpoint or command?
On `$e.data.get` the first parameter is a command.

Reply:
Commands support query as part of the command, the same as backend.

backend:
![image](https://user-images.githubusercontent.com/10234691/84654398-c60beb80-af17-11ea-9a1b-21136e26090c.png)

frontend:
![image](https://user-images.githubusercontent.com/10234691/84654437-db811580-af17-11ea-8951-4c70e0c8b2d2.png)